### PR TITLE
[BTFS-1086] - http-api-docs for btfs

### PR DESCRIPTION
**Pull request by uchetron**
Need to fork http-api-docs and make it work for BTFS!

### CODE REVIEW

**Comment by uchetron** [comment]
@taiyangc please review

File: endpoints.go [hunk 1]
```diff
@@ -59,16 +59,16 @@ func (a sorter) Less(i, j int) bool { return a[i].Name < a[j].Name }
 
 const APIPrefix = "/api/v0"
```
Comment (taiyangc) [inline endpoints.go]: We are up to "/api/v1" now, should probably pull this from https://github.com/TRON-US/go-btfs-api/blob/BTFS/request.go#L19 if possible?

File: endpoints.go [hunk 2]
```diff
@@ -1,16 +1,16 @@
-// Package docs can be used to gather go-ipfs commands and automatically
+// Package docs can be used to gather go-btfs commands and automatically
 // generate documentation or tests.
 package docs
 
 import (
 	"fmt"
+	config "github.com/TRON-US/go-btfs"
```
Comment (taiyangc) [inline endpoints.go]: Goes into the next import group

File: README.md [hunk 3]
```diff
@@ -2,11 +2,10 @@
 
 
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
-[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/project-BTFS-blue.svg?style=flat-square)](https://www.bittorrent.com/btfs/)
 [![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
```
Comment (taiyangc) [inline README.md]: could you remove all the badges? most don't apply to us.

File: README.md [hunk 4]
```diff
@@ -34,7 +33,7 @@ After installing you can run:
 > http-api-docs
 ```
 
-This should spit out a Markdown document. This is exactly the `api.md` documentation at https://github.com/ipfs/docs/blob/master/content/reference/api/http.md, so you can redirect the output to just overwrite that file.
+This should spit out a Markdown document. This is exactly the `api.md` documentation at https://github.com/TRON-US/go-btfs/tree/master/docs/content/reference/api/http.md, so you can redirect the output to just overwrite that file.
```
Comment (taiyangc) [inline README.md:32]: Update LICENSE section on the bottom as well.

Comment (taiyangc) [reply README.md:32]: Also the LICENSE file itself.

**Review by taiyangc — COMMENTED** [review]
Some minor change suggestions

**Comment by taiyangc** [comment]
@uchetron could you add me as admin on this repo? I need to make some branch rule changes. thx

File: endpoints.go (hunk 1 above)
Comment (uchetron) [reply endpoints.go]: ok.

File: endpoints.go (hunk 2 above)
Comment (uchetron) [reply endpoints.go]: ok

File: README.md (hunk 3 above)
Comment (uchetron) [reply README.md]: ok

**Comment by uchetron** [comment]
@taiyangc added you as an admin

File: README.md (hunk 4 above)
Comment (uchetron) [reply README.md:32]: @taiyangc not really sure what to update here...

Comment (taiyangc) [reply README.md:32]: Could you change `MIT (C) Protocol Labs, Inc.` to `MIT (C) TRON-US`? similarly in the LICENSE file

Comment (uchetron) [reply README.md:32]: ok.

File: LICENSE [hunk 5]
```diff
@@ -1,4 +1,4 @@
-MIT License
+MIT (C) TRON-US
```
Comment (taiyangc) [inline LICENSE]: For this file, could you change Line 3 instead? thanks.

Comment (uchetron) [reply LICENSE]: ok

**Review by taiyangc — APPROVED** [review]
lgtm

**Pull request merged**